ducktape/tests/reporter.py

```diff
@@ -224,7 +224,8 @@ class HTMLSummaryReporter(SummaryReporter):
     def format_test_name(self, result):
         lines = ["Module: " + result.module_name,
                  "Class:  " + result.cls_name,
-                 "Method: " + result.function_name]
+                 "Method: " + result.function_name,
+                 "Nodes:  " + str(result.nodes_used)]
```

Comment:
Nice!  Why not show the utilization as well to see if people are effectively using their nodes?
```suggestion
                 "Method: " + result.function_name,
                f "Nodes: {result.nodes_used}/{result.nodes_allocated}"]
```